Model Zoo API (#355)
onnx/models: model-zoo-api/README.md

# Python-API-Script
Onnx Python API Script used to download and save pretrained models from [onnx model zoo](https://github.com/onnx/models). Retrieves metadata after the model is successfully downloaded. 

## Features 

> ```get_model_versions(MODEL_FOLDER_NAME)``` - Retrieves an array of all the versions of the specificed model folder in the Onnx Model Zoo. 

Reviewer: Specify model_folder_name format here, perhaps with a default example.

> ```get_model_versions(MODEL_FOLDER_NAME)``` - Retrieves an array of all the versions of the specificed model folder in the Onnx Model Zoo. 

Reviewer: ```suggestion
```get_model_versions(MODEL_FOLDER_NAME)``` - Retrieves an array with strings values of all the versions of the specified model in the ONNX Model Zoo. 
```

   
```get_pretrained()``` - Downloads and saves specific onnx models to desired path.
            

```get_metadata()``` - Retrieves metadata of the onnx model. 
        
## Intialization and Usage
Initiate the object by calling onnx_zoo class name.
```
OBJECT_NAME = onnx_zoo()
```
The Python scipt will then ask to input the model folder name. When inputting the name, it should be in all lowercase. 

```
Enter Model Name: resnet
```
                       

> After model folder name is inputted, the script will output all the model versions that exist in the folder. 

Reviewer: ```suggestion
After model folder name is received from input, the script will display all the model versions that exist in the folder. 
```

```
['resnet101-v1-7', 'resnet101-v2-7', 'resnet152-v1-7', 'resnet152-v2-7', 'resnet18-v1-7', 'resnet18-v2-7', 'resnet34-v1-7', 'resnet34-v2-7', 'resnet50-caffe2-v1-3', 'resnet50-caffe2-v1-6', 'resnet50-caffe2-v1-7', 'resnet50-caffe2-v1-8', 'resnet50-caffe2-v1-9', 'resnet50-v1-7', 'resnet50-v2-7']
```

> From the array of versions, input the version that will be downloaded and input the local directory that the model will be saved in.

Reviewer: ```suggestion
The CLI script then prompts the user to input the version (from the array of model versions) that will be downloaded and input the local directory that the model will be saved in.
```

``` 
Enter model name from options: resnet101-v1-7 

> Enter saved path: /Users/name/Downloads

Reviewer: I feel that this "enter saved path" is not clear. Perhaps something like "Enter output directory name to save model:"
